scripts/all-command-help.js

```diff
@@ -4,99 +4,99 @@
 // Reads from dist directory, so be sure to build before running this target
 //
 
-'use strict'
-
-const _ = require('lodash')
-const fs = require('fs')
-const os = require('os')
-const path = require('path')
-const util = require('util')
-const Table = require('cli-table3')
-const { getClassHelpText, getOptionsDescription, getPositionalOptionsDescription } = require('../dist/util/commandline/option-decorators')
-const { getOptionsForTwoColumnTableWithNoBorders } = require('../dist/util/interaction/out')
+"use strict";
+
+const _ = require("lodash");
+const fs = require("fs");
+const os = require("os");
+const path = require("path");
+const util = require("util");
+const Table = require("cli-table3");
+const {
+  getClassHelpText,
+  getOptionsDescription,
+  getPositionalOptionsDescription,
+} = require("../dist/util/commandline/option-decorators");
+const { getOptionsForTwoColumnTableWithNoBorders } = require("../dist/util/interaction/out");
```

Comment:
maybe import here as well?

Reply:
Not touching the import syntax of .js files.